components/skeleton/skeleton.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import { SkeletonAnimations, SkeletonVariants } from '../utils/prop-types'
+
+interface SkeletonBaseProps {
+  animation?: SkeletonAnimations | boolean
```

Comment:
maybe the variable 'none' is better than the boolean

Reply:
agree